scripts/competitions/svcomp/esbmc-wrapper.py

```diff
@@ -292,7 +289,7 @@ def verify(strat, prop, fp_mode):
 property_file = args.propertyfile
 benchmark = args.benchmark
 strategy = args.strategy
-concurrency = args.concurrency
+concurrency = True if args.concurrency else False
```

Comment:
Is this really needed? Doesn't argparse return `false` (or `None`) when an option is not set?